docs/en/toolkit/frameworks.md

```diff
@@ -7,10 +7,12 @@ lang: en
 
 # Toolkit Frameworks
 
-Toolkit frameworks are collections of code that can be used across apps and engines. The Shotgun Utilities and QT Widgets frameworks are especially useful in app development.
+Toolkit frameworks are collections of code that can be used across apps and engines. You can read more about the frameworks in general [in our core API documentation](https://developer.shotgunsoftware.com/tk-core/platform.html#frameworks).
 
-The **Shotgun Utilities Framework**, or `tk-framework-shotgunutils`, is a collection of Shotgun-related helpers and utilities that make App development easier. It includes a Shotgun (MVC) Model, which makes it easy to connect a Shotgun query with a standard QT View, as well as classes for managing settings, background Shotgun processing, and globals such as schema, statuses, and icons.
+We also provide some pre made frameworks which you can use in your custom apps or app hooks.
```

Comment:
I'd make three small changes to this sentence:

We provide some pre-made frameworks, which you can use in your custom apps or app hooks.

(remove also, add hyphen, add comma)